### ISSUE TITLE
Implement getGroundDistance(BWAPI::TilePosition pos1, BWAPI::TilePosition pos2);

### ISSUE THREAD

**Issue by krasi0** [bwapi/bwapi#634]
The method should be similar to getDistance() but used when the AI is dealing with ground units.

**Comment by ZerGreenOne** [comment]
What algorithm would it use to implement this?

**Comment by heinermann** [comment]
The algorithm would use whatever Starcraft uses. That is a A* pathing through the region data, and then some kind of pathing refinement.

**Comment by krasi0** [comment]
Yes, it should directly use what  the StarCraft engine currently uses (the
fast approximation). If a bot author needs a more accurate A*
implementation, they could always use BWTA or implement the algorithm
themselves.
